### DISCUSSION TITLE
Swipe within the content area on the Tab Component on Mobile View

### DISCUSSION THREAD

**Discussion by zeshhaan** [element-plus/element-plus#9657] [Ideas]
I was looking for a component library that has swipable tabs but none of them seems to have it. It provides great experience on mobile matching the native feel and would be great to have!
Examples: 
https://material.io/components/tabs#behavior
https://github.com/HJ29/vue3-tabs
https://github.com/lpeihan/vue-slide-tabs

https://user-images.githubusercontent.com/34814964/188566631-de2bb3ff-2e58-4425-a544-71d07b1dfa9b.mp4

**Comment by teckhan** [comment]
out of the three i like [vue3-tabs](https://github.com/HJ29/vue3-tabs) most, due to not using absolute and transform, and output as a fragment.

however, this library is catching movement and acceleration of mouse down event, then perform the transition.

i think a better mobile experience will be the transition keeping up where your finger goes, which is supported native scroll snap, still prefer a custom one if feasible. 😂
